win32.tup

```diff
@@ -43,6 +43,10 @@ CFLAGS += -D_UNICODE
 
 CFLAGS32 += $(CFLAGS)
 
+LDFLAGS += -static-libgcc
+
+LDFLAGS32 += $(LDFLAGS)
```

Comment:
I had to change this to use $(LDFLAGS) in the rule rather than adding them to LDFLAGS32 here because it wasn't getting picked up in Tupfiles where we add to LDFLAGS after include_rules.